docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,966 @@
+  Name    Description    Physical-Slot    Oper-Status    Admin-Status    Serial
+------  -------------  ---------------  -------------  --------------  --------
+  DPU0            N/A               -1         Online              up       N/A
+  DPU1            N/A               -1         Online              up       N/A
+  DPU2            N/A               -1         Online              up       N/A
+  DPU3            N/A               -1         Online              up       N/A
+  DPU4            N/A               -1         Online              up       N/A
+  DPU5            N/A               -1         Online              up       N/A
+  DPU6            N/A               -1         Online              up       N/A
+  DPU7            N/A               -1         Online              up       N/A
+
+
+```
+#### Pass/Fail Criteria
+ * Verify DPU offline in show chassis modules status after DPU shut down
+ * Verify DPU is shown in show chassis modules status after DPU powered on
+
+
+### 1.7 Check removal of pcie link between NPU and DPU
+
+#### Steps
+ * Use command `pcieutil generate` to generate pcie yaml
+ * Use `show platform pcieinfo -c` to run the pcie info test to check everything is passing
+ * Use command `echo 1 > /sys/bus/pci/devices/BUS_ID/remove` to remove pcie link between NPU and one DPU
+ * Use `show platform pcieinfo -c` to run the pcie info test to check pcie link has been removed
+ * Use command `echo 1 > /sys/bus/pci/rescan` to rescan pcie links
+ * Use `show platform pcieinfo -c` to run the pcie info test to check everything is passing
+ * This test is to check the PCie hot plug functinality since there is no OIR possible
+
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch: Showing example of one DPU pcie link
+
+root@sonic:/home/cisco# pcieutil generate
+Are you sure to overwrite config file pcie.yaml with current pcie device info? [y/N]: y
+Generated config file '/usr/share/sonic/device/x86_64-8102_28fh_dpu_o-r0/pcie.yaml'
+root@sonic:/home/cisco# show platform pcieinfo -c
+
+root@sonic:/home/cisco# echo 1 > /sys/bus/pci/devices/0000:1a:00.0/remove
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# echo 1 > /sys/bus/pci/rescan
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# show platform pcieinfo -c
+
+```
+#### Pass/Fail Criteria
+ * Verify after removing pcie link, pcie info test fail only for that pcie link.
+ * Verify pcieinfo test pass for all after bringing back up the link
+
+
+### 1.8 Check the NTP date and timezone between DPU and NPU
+
+#### Steps
+ * Use command `date` to get date and time zone on Switch
+ * Use command `ssh admin@169.254.x.x` to enter into required dpu.
+ * Use command `date` to get date and time zone on DPU
+   
+#### Verify in
+ * Switch and DPU
+   
+#### Sample Output
+```
+On Switch:
+
+root@sonic:/home/cisco# date
+Tue 23 Apr 2024 11:46:47 PM UTC
+root@sonic:/home/cisco#
+.
+root@sonic:/home/cisco# ssh admin@169.254.24.1
```

Comment:
The IP network of the DPU is incorrect. Please align with https://github.com/sonic-net/SONiC/blob/master/doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md document

Reply:
Changed it.